src/js/constants/PodContainerState.js

```diff
@@ -0,0 +1,14 @@
+//
```

Comment:
Done. (I usually use `/** .. */` for jsdoc comments, this particular case was just notes)